drivers/input/Kconfig.kpp

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2025 Google, LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config INPUT_KPP
```

Comment:
you add a Kconfig here which is referenced in previous commits before this commit